src/factors/Pose2D.jl

```diff
@@ -42,7 +42,20 @@ end
 Pose2Pose2(::UniformScaling) = Pose2Pose2() # MvNormal(zeros(3),LinearAlgebra.diagm([1.0;1.0;1.0])) )
 
 
-getSample(cf::CalcFactor{<:Pose2Pose2}, N::Int=1) = (rand(cf.factor.z,N), )
+function getSample(cf::CalcFactor{<:Pose2Pose2}, N::Int=1) 
+  Z = cfo.factor.Z
+  M = getManifold(Pose2)
+  p = getPointIdentity(Pose2)
+  
+  Xc = [rand(Z) for _ in 1:N]
+  
+  # X = get_vector.(Ref(M), Ref(p), Xc, Ref(DefaultOrthogonalBasis()))
+  X = hat.(Ref(M), Ref(p), Xc)
+  points = exp.(Ref(M), Ref(p), X)
+
+  (points, )
+end
```

Comment:
We should just synchronize on how we are going to work with relative factors. I used a measurement on the tangent at p.
or is this just a step in that direction

Reply:
we should probably leave that up to the user and not try standardize if not needed.  I'm assuming the user wants to pass to the residual factor function various data, which is currently `Tuple of Vectors[sample_idx]`.  Only the user will be using the measurement values in AbstractRelative.

It's a little harder with AbstractPrior which will likely have to be points rather than tangents at this time.

Reply:
Ok, Then maybe just for parametric? As you know parametric doesn't use `getSample`.
I would say the default is on the tangent space and if you want to use a point, you should write a custom getParamametricMeasruemt